transmogrifier.dart

```diff
@@ -156,6 +156,9 @@ Future<Function> getSinkFunction(String name) async {
 }
 
 Future<String> getSchema(String path) async {
+  if (path == "") {
```

Comment:
should this check for null/undefined, etc, if so you can use: if (!path)

Reply:
probably a good idea, will add

Reply:
actually im not sure the above would work for Dart. Im still a little rocky on Dart so I think keeping it as is is probably the safest bet for now